### DISCUSSION TITLE
Interstitial.load crash and close App

### DISCUSSION THREAD

**Discussion by RayosElDev** [invertase/react-native-firebase#5166] [General]
Hello, I just follow the tutorial to add Admob in my react App.

Analytics works fine.
Admob Banner works fine

But when I just copy the example of Interstitial or Reward Ad I have a crash when I call the line interstitial.load();

the code is the same of here:
https://rnfirebase.io/admob/usage
`import React, { useEffect, useState } from 'react';
import { Button } from 'react-native';
import { InterstitialAd, AdEventType, TestIds } from '@react-native-firebase/admob';

const adUnitId = __DEV__ ? TestIds.INTERSTITIAL : 'ca-app-pub-xxxxxxxxxxxxx/yyyyyyyyyyyyyy';

const interstitial = InterstitialAd.createForAdRequest(adUnitId, {
  requestNonPersonalizedAdsOnly: true,
  keywords: ['fashion', 'clothing'],
});

function App() {
  const [loaded, setLoaded] = useState(false);

  useEffect(() => {
    const eventListener = interstitial.onAdEvent(type => {
      if (type === AdEventType.LOADED) {
        setLoaded(true);
      }
    });

    // Start loading the interstitial straight away
    interstitial.load();

    // Unsubscribe from events on unmount
    return () => {
      eventListener();
    };
  }, []);

  // No advert ready to show yet
  if (!loaded) {
    return null;
  }

  return (
    <Button
      title="Show Interstitial"
      onPress={() => {
        interstitial.show();
      }}
    />
  );
}`

**Comment by mikehardy** [comment]
Duplicate of #5093 and I have a reproduction there, hoping to give it a go tomorrow

**Reply by RayosElDev** [reply]
Hello, thanks for wauick answer.
No, it's not the same..
I used the example code not my custom code
And he is working in Android..
I have a problem in Android...
I checked this post  before
He with example code, can Load interstitial
Me not..always crash in Load process.

I just can show banner

**Reply by RayosElDev** [reply]
Would you update a project example to download and run?
I follow all steps like web and you vídeo :S

**Comment by Tarikk01** [comment]
workaround
app/build.gradle

    implementation("com.google.android.gms:play-services-ads:19.8.0") { force = true; }

**Reply by RayosElDev** [reply]
Oh Seems works!
